### DISCUSSION TITLE
Found while cleaning hard drive

### DISCUSSION THREAD

**Discussion by pixelwaster** [dbuezas/lgt8fx#201] [Show and tell]
![lgt3](https://user-images.githubusercontent.com/50377597/157385824-4cebe930-8b37-449b-8976-4fa07e8f2327.jpg)
 If you look closely at the crystal, it is 50.00 Mhz. It was overclocked and ran the blink sketch. I've got some fundamental 50 MHz xtals in my crystal/osc/vfo/channel element cabinet. I wonder...

**Comment by pixelwaster** [comment]
[LGTSDK_Builder_1.5beta29.zip](https://github.com/dbuezas/lgt8fx/files/8212343/LGTSDK_Builder_1.5beta29.zip)
[test.zip](https://github.com/dbuezas/lgt8fx/files/8212344/test.zip)
This is something I've only played with once. test.zip is the result. I set some pins as inputs, outputs, setup AC0 and AC1, and configured the USART. It spit out main.c, all the .h and .c files I needed. Just needed to add my code.

**Comment by pixelwaster** [comment]
[LGT8F328P computing accelerator reference code.zip](https://github.com/dbuezas/lgt8fx/files/8212414/LGT8F328P.computing.accelerator.reference.code.zip)
[LGT8F328P differential amplifier reference code.zip](https://github.com/dbuezas/lgt8fx/files/8212415/LGT8F328P.differential.amplifier.reference.code.zip)
[LGT8F328P peripheral reference code.zip](https://github.com/dbuezas/lgt8fx/files/8212416/LGT8F328P.peripheral.reference.code.zip)

Last three. These have sample code to demo bits of the mcu. The adc_bench in peripherals looked interesting. 
The computing accelerator was full of math routines.
The differential amplifier reference  had this in main.c 

```
============================================ 
    LGT8F328P Software Developement Kits 
	**** ADC/DAP Module Usage **** 
============================================ 

	// enable ADC's test mode to drive
	// ADC's S/H channel to PE6 (for test purpose)
	// for this case, DAP's output can be measured directly from PE6  <------------did I miss this on the diagram??
```

**Reply by gadefox** [reply]
FIY: `sq` is not working..
sq(200)
4294941760
sq(300)
24464
